httpx/config.py

```diff
@@ -166,6 +168,11 @@ def _create_default_ssl_context(
         if ssl.HAS_NPN:  # pragma: no cover
             context.set_npn_protocols(http_versions.alpn_identifiers)
 
+        if hasattr(context, "keylog_filename"):
+            keylogfile = os.environ.get("SSLKEYLOGFILE")
+            if keylogfile and not sys.flags.ignore_environment:
+                context.keylog_filename = keylogfile  # type: ignore
```

Comment:
What happens if the specified file name points at a non-existing location? Will Python create it for us?

Reply:
If it's a non-existing file, Python creates it. However, if it's a path that contains a non-existing directory it will raise `FileNotFoundError`.